tests/unit/test_discovery_client.py

```diff
@@ -223,7 +228,7 @@ def test___get_discovery_service_address___start_service_jit___returns_expected_
     assert _TEST_SERVICE_PORT in discovery_service_address
 
 
-def test___get_discovery_service_address___key_file_not_exist___throws_timeouterror(
+def test___get_discovery_service_address___open_key_file_throws_ioerror___throws_timeouterror(
```

Comment:
This test name and test___get_discovery_service_address___start_service_jit___returns_expected_value have the given and when clauses swapped.

Reply:
https://github.com/ni/measurementlink-python/pull/404